skbio/io/format/embl.py

```diff
@@ -1352,17 +1320,27 @@ def _serialize_sequence(obj, indent=5):
     obj : DNA, RNA, Sequence Obj
     '''
 
-    # magic numbers
-    n = 1
-    line_size = 60
+    # a flag to determine if I wrote header or not
+    flag_header = False
+
+    # magic numbers: there will be 60 letters (AA, bp) on each line
+    chunk_size = 60
+
+    # letters (AA, bp) will be grouped by 10: each group is divided by
+    # one space from each other
     frag_size = 10
 
-    # count bases in sequence
+    # this is the final line length
+    line_size = 80
+
+    # get sequence as a string with lower letters (uniprot will be upper!)
     seq = str(obj).lower()
-    n_a = seq.count("a")
-    n_c = seq.count("c")
-    n_g = seq.count("g")
-    n_t = seq.count("t")
+
+    # count bases in sequence
+    n_a = sum(obj.frequencies({"a", "A"}).values())
```

Comment:
`DNA` or `RNA` store all capital case. no need to use both. suggesting doing this:

````
freq = obj.frequencies()
n_a = freq.pop('A')
...
n_others = sum(freq.values())
````